### PR TITLE
Update Suffixes

### PR DESCRIPTION
**Pull request by DaveMcCormack**
Find updates for suffixes.

Thank you.

### CODE REVIEW

**Comment by weppos** [comment]
Please read https://github.com/publicsuffix/list/wiki/Guidelines and update the request accordingly.

**Comment by DaveMcCormack** [comment]
What is missing from the submission?

I have read the 'Guidelines', and believe that I have submitted correctly.

Thank you.

**Comment by DaveMcCormack** [comment]
OK.

I have gone through the 'Guidelines' multiple times, and reviewed all steps involved.

I believe the submission to be in the correct format, etc.

Can you explain what the issue is, so that whatever needs to b done can be done?

Thank you.

**Comment by DaveMcCormack** [comment]
OK.

I have reviewed the 'Guidelines', and the submission.

a) The submission is in the correct format, and passes the 'Travis' check.
b) All suffixes submitted have a '_psl' 'txt' DNS entry, pointing to the 'GitHub' pull request.

Can you update this thread, and explain any outstanding issues, so that these can be rectified promptly?

Thank you.

**Comment by DaveMcCormack** [comment]
I do not understand how this 'GitHub' system works.

Straight out. I have no idea.

A bit of feedback explaining what the issue is with my submission would be nice.

Thank you.

**Comment by DaveMcCormack** [comment]
OK.

I have once again reviewed the 'Guidelines'.

a) 'NymNom' is a private domain name registry, that offers users the ability to register domains in the listed suffixes.
b) The patch submitted is in the correct format, and passes the 'Travis' check.
c) All suffixes submitted have a '_psl' 'txt' DNS entry, pointing to the 'GitHub' pull request.

Please, provide feedback on what the issue is, so that it can be rectified promptly.

Please.  That is not too much to ask.

Thank you.

**Comment by weppos** [comment]
Dave, please relax. Personally, I've been travelling these days, and I cannot provide real-time feedback.

**Comment by sleevi** [comment]
@DaveMcCormack Please provide example domains and the rationale for why you're requesting inclusion.

Thanks!

**Comment by DaveMcCormack** [comment]
Hello,

Thanks for the reply.

The rationale for inclusion -

'NymNom' is a private domain name registry (PDNR), providing domain name registration, based on submitted suffixes, to 3rd parties.
These are fully delegated domain names, with 'SOA' (et al) records (full delegation).
The service allows 3rd parties to register and use domain names based on the suffixes supplied.
The registry 'NymNom' wishes to be included in the list, as it is a domain name registry.
Inclusion in the list, will allow delgated, 3rd party registered, domains to be considered separate from the parent suffix.

In short, 'NymNom' is a private domain name registry (PDNR), providing fully delegated domain names to 3rd parties.


Example of domain names in use (fully delegated, registered to 3rd parties) -

wikipedia.nom.mk
wiktionary.nym.mx
wikiquote.nom.cl
wikinews.nym.pe
wikisource.nom.al


These are live domain names, that resolve via http.

Also, run a DNS lookup, which will demonstrate 'SOA' (et al) records for all domains, demonstrating full delegation.


Thank you for your time,

**Comment by sleevi** [comment]
@DaveMcCormack Is http://nymnom.com/ the correct URL for more information? I just get a blank page in the browser for both it and www.nymnom.com. For example, how is such a domain registered, and where is the list?

Also, can you please ensure your company name maintains the existing alphabetical sort?

**Comment by DaveMcCormack** [comment]
Hello,

Thanks for the reply.

Q. - Is http://nymnom.com/ the correct URL for more information? I just get a blank page in the browser for both it and www.nymnom.com. For example, how is such a domain registered, and where is the list?

The main domain hasn't been available to the general public, as NN is finishing the beta-testing before launch to GP.
The submission to the list is part of the finalization details before GP launch
You can find a registration portal at - https://shop.nymnom.com/
This is the portal, used as a branch of the main 'NymNom' domain.
There it should have a list of all suffixes submitted.

Q. - Also, can you please ensure your company name maintains the existing alphabetical sort?

I think the answer to that question is the following -

Do a 'whois' lookup on nymnom.com and any of the submitted suffixes.
Compare the results of the registrant of the domains queried.  
There will be able to see that the registrant of nymnom.com is the same company who has registered for the various suffixes submitted.
This should demonstrate that the company behind nymnom.com is the same as the various suffixes, thus the company maintains and is the ultimate authority for these suffixes.
'NymNom' is the registry and ultimate provider of these domains.


Thank you for your time,

**Comment by sleevi** [comment]
> I think the answer to that question is the following 

No. It doesn't. Your entry appears in the wrong place in the public suffix list. It's sorted by company name.

**Comment by DaveMcCormack** [comment]
Hello,

Thanks for your reply.

I have updated the submission, so as to be alphabetical by company name, when sorted.

This addresses your previous reply.

Please indicate if there are any outstanding tasks to perform.

Thank you for your time,

**Comment by DaveMcCormack** [comment]
Hello,

A quick question -

The change listed above -

'Remove unrelated change			2169c3e'

I don't understand what happened there.

I am new to this 'GitHub' system, so to ask a stupid question, what was the issue there?

That way I can learn what I did wrong, when using the 'GitHub' system.

Thank you for your time,

**Comment by sleevi** [comment]
You may have just typo'd an extra line :)

**Comment by DaveMcCormack** [comment]
Hello,

I was wondering when this pull request will be merged with the main branch?

When is that scheduled to happen?

Thank you for your time,

**Comment by DaveMcCormack** [comment]
Hello,

I was wondering if I can get some feedback on when the submission will be merged?

Is there any other tasks or information required?

Could I get some indication on when the submission will be merged?

Thank you for your time,

**Comment by sleevi** [comment]
This list is maintained voluntarily, as time permits. Unfortunately, no
estimations can be provided, as it varies with volunteer time.

**Comment by DaveMcCormack** [comment]
Hello,

Thank you for your reply.

Can I get the 'waiting-followup' label removed from this submission?

It occurs to me that it is inaccurate, as there is no follow up waiting.
Please correct me if I'm wrong.

Also, it might be interpreted that the submission is not ready to be processed for merging, if the 'waiting-followup' label is present.

Thank you for your time,

File: public_suffix_list.dat [hunk 1]
```diff
@@ -11667,6 +11667,46 @@ zapto.org
...
 nyc.mn
 
+// NymNom : http://nymnom.com/
+// Submitted by Dave McCormack <dave.mccormack@nymnom.com>
+nom.ae
+nom.ai
+nom.al
+nom.cl
+nom.gd
+nom.gl
```
Comment (sleevi) [inline public_suffix_list.dat]: Unable to validate this domain

File: public_suffix_list.dat [hunk 2]
```diff
@@ -11667,6 +11667,46 @@ zapto.org
...
+nym.by
+nym.bz
+nym.gr
+nym.kz
+nym.la
+nym.li
+nym.lt
+nym.lu
+nym.me
+nym.mx
```
Comment (sleevi) [inline public_suffix_list.dat]: Unable to validate this domain

File: public_suffix_list.dat [hunk 3]
```diff
@@ -11667,6 +11667,46 @@ zapto.org
...
+nom.nu
+nom.pw
+nom.qa
+nom.rs
+nom.si
+nom.ug
+nom.uy
+nom.vc
+nom.vg
+nym.by
```
Comment (sleevi) [inline public_suffix_list.dat]: These should be sorted by the right-most label (the TLD) to the left-most label (nom vs nym); it's presently sorted left-most to right-most.

**Comment by DaveMcCormack** [comment]
Hello,

QUOTE - "These should be sorted by the right-most label (the TLD) to the left-most label (nom vs nym); it's presently sorted left-most to right-most."

I have updated the submission.

I have ordered the suffixes, according to their TLD order.

I believe that is the meaning and solution to your comment.

Please advise if other.

Look forward to the submission being completed and merged.

Thank you for your time,

**Comment by DaveMcCormack** [comment]
Hello,

I have updated the submission.

I have ordered the suffixes, according to their TLD order.

I believe that is the meaning and solution to your comment.

Please advise if other.

Look forward to the submission being completed and merged.

Thank you for your time,


sleevi wrote:
> These should be sorted by the right-most label (the TLD) to the 
> left-most label (nom vs nym); it's presently sorted left-most to 
> right-most.

**Comment by sleevi** [comment]
The following hosts are missing their validation information:
nym.mx
nom.gl

**Comment by DaveMcCormack** [comment]
Hello,

Thank you for your email and reply.

Please find the validation information corrected and updated at -

nym.mx
nom.gl

Please let me know if there is any other information needed or tasks to 
be completed.

I look forward to the timely completion of this submission being 
accepted.  It has been in the process since March 2017.

This must be quite soon, as there surely can't be any more issues.

Thank you for your time,




sleevi wrote:

**Comment by sleevi** [comment]
The speed at which requests can be processed is measured by compliance with https://github.com/publicsuffix/list/wiki/Guidelines#submit-the-change

As it stands, after querying both `ns1.nym.mx` and `ns2.nym.mx`, the TXT record for `_psl.nym.mx` is still not published.

**Comment by DaveMcCormack** [comment]
Hello,

Thank you for your reply,

nslookup, using Google NS system

nslookup -q=txt _psl.nom.gl 8.8.8.8
nslookup -q=txt _psl.nym.mx 8.8.8.8

Both return validation reference.

If there are any further whatevers, let me know.

Thank you for your time,




sleevi wrote:

**Comment by DaveMcCormack** [comment]
Hello,

Thank you.

I appreciate it.

Thank you for your time,

Dave McCormack


sleevi wrote:
